src/inc/msquic.h

```diff
@@ -539,9 +539,9 @@ typedef enum QUIC_PERFORMANCE_COUNTERS {
 #ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
 typedef struct QUIC_VERSION_SETTINGS {
 
-    uint32_t* AcceptableVersions;
-    uint32_t* OfferedVersions;
-    uint32_t* FullyDeployedVersions;
+    const uint32_t* AcceptableVersions;
```

Comment:
I wanted to make this change back when I did the version changes, but I was uneasy about the all the casting away the `const` internally.